### DISCUSSION TITLE
Complejidad temporal de codigo recursivo

### DISCUSSION THREAD

**Discussion by SantiagoBravo05** [INGCOM-UNRN-P1/cursada-2023#220] [Preguntas y respuestas] [answered]
No se bien como hacer la complejidad temporal de los ejercicios recursivos y no encontre informacion parecida a la que estamos viendo en clase.

**Comment by inomdedeu** [comment, marked as answer]
Tenes que contas el costo computacional de las instrucciones de ese llamado. 
Y sumarles el costo del nuevo llamado

![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/86799661/c1fd30ef-0e70-45fb-b52e-ec18d7e7abe8)
